components/kyma-environment-broker/cmd/broker/broker_suite_test.go

```diff
@@ -373,7 +373,7 @@ func (s *BrokerSuiteTest) WaitForProvisioningState(operationID string, state dom
 
 func (s *BrokerSuiteTest) WaitForOperationState(operationID string, state domain.LastOperationState) {
 	var op *internal.Operation
-	err := wait.PollImmediate(pollingInterval, 20*time.Second, func() (done bool, err error) {
+	err := wait.PollImmediate(pollingInterval, 2*time.Second, func() (done bool, err error) {
```

Comment:
the increased polling timeout for tests done in https://github.com/kyma-project/control-plane/pull/1751/commits/83837849c96d1b619db5ae8119043718ac41e8ef is no longer necessary after https://github.com/kyma-project/control-plane/pull/1773